lms/static/sass/features/_journals.scss

```diff
@@ -0,0 +1,169 @@
+// journal catalog listing
```

Comment:
Does this sass file need to be in a centralized place in the platform or can it be located within the journals app?
FYI @arizzitano